src/bentoml/_internal/bento/build_config.py

```diff
@@ -705,6 +705,38 @@ def _converter(value: OptionsCls | dict[str, t.Any]) -> options_type:
     return _converter
 
 
+@attr.frozen
+class ModelSpec:
+    tag: str
+    filter: t.Optional[str] = None
```

Comment:
can filter be a list? We may have multiple filters

Reply:
By design it is the same as the "filter" input on BentoCloud, where you can connect multiple filters by ","

Reply:
that's great